src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$cell-size: 100px;
+$cell-indent: 1px;
```

Comment:
Consider renaming $cell-indent to $calendar-cell-gap to better represent its purpose in the context of the task.